src/content/docs/apm/agents/net-agent/other-installation/understanding-net-agent-environment-variables.mdx

```diff
@@ -0,0 +1,211 @@
+---
+title: Understanding required .NET agent environment variables
+tags:
+  - Agents
+  - NET agent
+  - Other installation
+redirects:
+---
+
+Our .NET agent uses environment variables to attach the profiler to your application, which allows the .NET agent and profiler to find required assets. If you're interested in diving deeper into how environment variables are used, see the sections below.
+
+## Terms overview
+
+Before reading the sections below, here are some tips for understanding terms used on this page:
+
+* ".NET Core" refers to .NET versions (and applications built with and for .NET versions) 5.0 and higher (just ".NET") as well as .NET Core 2.0 through 3.1.
+* "Profiler" refers to the component of the agent that implements the .NET profiling API. When things are configured correctly, the CLR attaches the profiler to your .NET application. The profiler then loads the rest of the agent into the application. 
+
+## How environment variables are used by the .NET agent
+
+Our .NET agent is a [.NET CLR profiler](https://learn.microsoft.com/en-us/dotnet/framework/unmanaged-api/profiling/profiling-overview) and implements a subset of the .NET profiling API.
+[Certain environment variables](https://learn.microsoft.com/en-us/dotnet/framework/unmanaged-api/profiling/setting-up-a-profiling-environment) must be present in the environment of a .NET process to tell the .NET runtime that the process should be profiled, and by what profiler.
+Additional environment variables are used by the .NET agent itself to let it know where and how it is installed.
+
+
+## What environment variables are required to use the .NET agent
```

Comment:
@nr-ahemsath It's unclear which variables are required? There should be content here, like a table or code sample listing out all the required ones.

Reply:
The rest of the document explains what variables are required under what circumstances.  It's too complicated for a single table or code sample.

Reply:
Ah okay, then it just needs a preface to the sub-sections below. I'll add that now.